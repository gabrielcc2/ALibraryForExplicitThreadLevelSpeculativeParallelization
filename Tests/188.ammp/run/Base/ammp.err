==5865== exp-bbv, a SimPoint basic block vector generator
==5865== NOTE: This is an Experimental-Class Valgrind Tool
==5865== Copyright (C) 2006-2010 Vince Weaver
==5865== Using Valgrind-3.6.1 and LibVEX; rerun with -h for copyright info
==5865== Command: ../Base/ammp_parallel
==5865== 
==5865== 
==5865== 
==5865== 
==5865== # Thread 1
==5865== #   Total intervals: 213 (Interval Size 100000000)
==5865== #   Total instructions: 21352591536
==5865== #   Total reps: 15117655
==5865== #   Unique reps: 13596678
==5865== #   Total fldcw instructions: 7594698
==5865== 
==5865== 
==5865== 
==5865== 
==5865== # Thread 2
==5865== #   Total intervals: 1458 (Interval Size 100000000)
==5865== #   Total instructions: 145817223080
==5865== #   Total reps: 0
==5865== #   Unique reps: 0
==5865== #   Total fldcw instructions: 1667384
==5865== 
==5865== 
==5865== 
==5865== 
==5865== # Thread 3
==5865== #   Total intervals: 930 (Interval Size 100000000)
==5865== #   Total instructions: 93031008883
==5865== #   Total reps: 0
==5865== #   Unique reps: 0
==5865== #   Total fldcw instructions: 1667384
==5865== 
==5865== 
==5865== 
==5865== 
==5865== # Thread 4
==5865== #   Total intervals: 653 (Interval Size 100000000)
==5865== #   Total instructions: 65345628072
==5865== #   Total reps: 0
==5865== #   Unique reps: 0
==5865== #   Total fldcw instructions: 1667616
==5865== 
==5865== 
==5865== 
==5865== 
==5865== # Thread 5
==5865== #   Total intervals: 472 (Interval Size 100000000)
==5865== #   Total instructions: 47218679589
==5865== #   Total reps: 0
==5865== #   Unique reps: 0
==5865== #   Total fldcw instructions: 1668776
==5865== 
==5865== 
