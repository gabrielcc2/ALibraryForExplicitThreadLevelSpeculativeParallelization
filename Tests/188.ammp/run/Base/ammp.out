echo off;
 setf mxdq 0.75;
  setf mmbox 10.;
 setf temp 300;
 seti numstp 333;
 setf bbox 35.;
 use none box bond angle hybrid torsion nonbon;
  echo off;
 tpac numstp .00001 temp;
 monitor;
 0.000000 unknown potential type
 351.914795 bond energy
 1002.165955 angle energy
 33.156345 hybrid energy
 419.105652 torsion energy
 6466.528320 non-bonded energy
 8272.871094 total potential energy
 8661.678711 total kinetic energy
 16934.549805 total energy
 388.807617 total action
  echo off;
 tpac numstp .00001 temp;
 monitor;
 0.000000 unknown potential type
 354.652832 bond energy
 1042.761230 angle energy
 33.098850 hybrid energy
 437.150330 torsion energy
 6120.482422 non-bonded energy
 7988.145508 total potential energy
 8684.481445 total kinetic energy
 16672.626953 total energy
 696.335938 total action
  echo off;
 tpac numstp .00001 temp;
 monitor;
 0.000000 unknown potential type
 356.007233 bond energy
 1064.004272 angle energy
 34.828671 hybrid energy
 438.652802 torsion energy
 5952.687500 non-bonded energy
 7846.180664 total potential energy
 8666.120117 total kinetic energy
 16512.300781 total energy
 819.939453 total action
  echo;
 exit;
